README.md

```diff
@@ -1,9 +1,91 @@
-Publish your digital garden / obsidian vault / markdown wiki / dataset in seconds. For free. No coding required.
+# Flowershow 🌷
+> Publish your Obsidian notes, beautifully. For free, no coding.
 
-Status: 🏗️ under very active construction 👷‍♀️
+Flowershow is an open-source tool for easily converting your markdown files into an elegant website. It's built on a standard, modern web stack – React, Next.js, and Tailwind and shipped with a basic default theme to get you started with just a few clicks.
 
-https://flowershow.app/
+- **Crafted for Obsidian**, Flowershow works with your Obsidian vault out of the box. No need to modify the syntax or change the file layout[^1].
+- **Elegant and functional** default theme with basic front page and navbar.
+- **Easy to tweak and extend** with custom page layouts and custom React components, that can be used within your markdown.
+- **Backed by a team of data geeks**, who love Markdown, and are passionate about sharing knowledge and ideas with others.
+
+[👉 See Flowershow in action!](https://flowershow.app/)
+
+[^1]: If you're not an Obisidan fan, Flowershow can be used with any CommonMark or GFM files.
```

Comment:
I think this is important enough i'd keep in the main body rather than a footnote. Plus footnote are rare for READMEs and academic-y in general.